### DISCUSSION TITLE
Is it possible to increase collisions?

### DISCUSSION THREAD

**Discussion by shiouhuamien** [detomon/wigglebone#16] [Ideas]
This physical skeleton plug-in is really great, simpler and more effective than the official physical skeleton.
Should become an official physical skeleton XD
But there is currently no collision, is it possible in the future?

**Comment by detomon** [comment]
Thanks, glad you like it.
Adding collision support is certainly possible. I will look into it for a future release.

**Comment by Lunrae** [comment]
I would love for that to be added as well and an eventual feature. godot released their own springbone but it's honestly not great compared to what you've accomplished here. What you did should be an official addition.
Collision support would be the icing on the cake here to make it perfect for a lot of different cases.

**Reply by shiouhuamien** [reply]
> SpringBone

Indeed, while the official SpringBone collision settings are very quick and convenient,
the physics simulation results are very unnatural and difficult to adjust.
Therefore, I still use this plugin where collisions are unnecessary.
I only use the official SpringBone when collisions are required.
